packages/frontend/core/src/desktop/pages/auth/magic-link.tsx

```diff
@@ -78,8 +79,16 @@ export const Component = () => {
           }
         });
       })
-      .catch(e => {
-        nav(`/sign-in?error=${encodeURIComponent(e.message)}`);
+      .catch(err => {
+        const userFriendlyError = UserFriendlyError.fromAnyError(err);
```

Comment:
Because this route is only valid for the web, the web is unlikely to trigger the minimum version restriction, so this may be useless

Reply:
Oops I seem to have made a basic mistake, I need to rethink this.